src/pages/User/components/PullRequests/PullRequest/index.js

```diff
@@ -1,20 +1,66 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import MergeStatus from './MergeStatus';
 import PullRequestInfo from './PullRequestInfo';
+import { isPullRequestMerged } from '../../../../../utils/utils';
 
 const ISSUE_STATUS = {
   ALL: 'all',
-  OPEN: 'open',
-  CLOSED: 'closed'
+  OPEN: 'open'
 };
 
-const PullRequest = ({ pullRequest }) => (
-  <div className={`bg-white leading-normal p-4 flex border-b border-grey break-words`}>
-    <MergeStatus open={pullRequest.state === ISSUE_STATUS.OPEN} merged={pullRequest.state === ISSUE_STATUS.CLOSED} />
-    <PullRequestInfo pullRequest={pullRequest} />
-  </div>
-);
```

Comment:
Hey @Vhndaree, at the second thought I'd recommend just leave this component stateless as before. You don't need state here. All you need to do is ask for a prop `isMerged` from the parent (right now you've declared it in the state which I don't think is needed). 

This will make things simpler. Also this will push your implementation to the upstream where there is already state management and API fetching logic - so let it be there and convert this to a functional (dump) component which only asks for one additional prop `isMerged`.

Reply:
Check the implementation of the parent component here: 
https://github.com/leapfrogtechnology/frogtoberfest/blob/e4b22a2b77b16072d945af89a53932d6f061f76e/src/pages/User/components/PullRequests/index.js#L257-L259

All you need to do in the parent component is to ensure that the `pullRequest` prop has `isMerged` key (boolean).

Reply:
@kabirbaidhya  Then should I fetch `isMerged` api from `PullRequests` component while fetching other apis and pass one extra props `isMerged` to `PullRequest` component.